huaweicloud/config/endpoints.go

```diff
@@ -16,27 +16,28 @@ type ServiceCatalog struct {
 // multiCatalogKeys is a map of primary and derived catalog keys for services with multiple clients.
 // If we add another version of a service client, don't forget to update it.
 var multiCatalogKeys = map[string][]string{
-	"iam":       {"identity", "iam_no_version"},
-	"bss":       {"bssv2"},
-	"ecs":       {"ecsv21", "ecsv11"},
-	"evs":       {"evsv21"},
-	"cce":       {"ccev1", "cce_addon"},
-	"cci":       {"cciv1_bata"},
-	"vpc":       {"networkv2", "vpcv3", "fwv2"},
-	"elb":       {"elbv2", "elbv3"},
-	"dns":       {"dns_region"},
-	"kms":       {"kmsv1"},
-	"mrs":       {"mrsv2"},
-	"rds":       {"rdsv1"},
-	"waf":       {"waf-dedicated"},
-	"geminidb":  {"geminidbv31"},
-	"dli":       {"dliv2"},
-	"dcs":       {"dcsv1"},
-	"dis":       {"disv3"},
-	"dms":       {"dmsv2"},
-	"dws":       {"dwsv2"},
-	"apig":      {"apigv2"},
-	"modelarts": {"modelartsv2"},
+	"iam":          {"identity", "iam_no_version"},
+	"bss":          {"bssv2"},
+	"ecs":          {"ecsv21", "ecsv11"},
+	"evs":          {"evsv21"},
+	"cce":          {"ccev1", "cce_addon"},
+	"cci":          {"cciv1_bata"},
+	"vpc":          {"networkv2", "vpcv3", "fwv2"},
+	"elb":          {"elbv2", "elbv3"},
+	"dns":          {"dns_region"},
+	"kms":          {"kmsv1"},
+	"mrs":          {"mrsv2"},
+	"rds":          {"rdsv1"},
+	"waf":          {"waf-dedicated"},
+	"geminidb":     {"geminidbv31"},
+	"dli":          {"dliv2"},
+	"dcs":          {"dcsv1"},
+	"dis":          {"disv3"},
+	"dms":          {"dmsv2"},
+	"dws":          {"dwsv2"},
+	"apig":         {"apigv2"},
+	"modelarts":    {"modelartsv2"},
+	"servicestage": {"servicestagev2"},
```

Comment:
why we have multi catalog here? seems we can just use one.

Reply:
the authorization resources use ver.1 API